cvat-ui/webpack.config.js

```diff
@@ -31,6 +31,7 @@ module.exports = (env) => ({
         host: process.env.CVAT_UI_HOST || 'localhost',
         port: 3000,
         historyApiFallback: true,
+        host: process.env.CVAT_UI_HOST || 'localhost',
```

Comment:
@dvkruchinin , do we need the line two times?

Reply:
Unfortunately, I am not the author of this change.
@ActiveChooN 
Can you please tell if it should be like this? If not, I will delete the extra line.

Reply:
OK. I will merge to unblock the PR. Thanks.